### PR TITLE
added intergration tests

### PR DESCRIPTION
**Pull request by businesskaniki**
Used Capybara to write integration tests for each view in your project.

### CODE REVIEW

**Review by andersonlebon — CHANGES_REQUESTED** [review]
### Hi @businesskaniki and @TahenyBELGUITH 
# Required Changes ♻️
Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

File: spec/intergration/post_index_spec.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts Page', type: :view do
+  describe 'posts#index page' do
+    before(:each) do
+      @user1 = User.create(name: 'Sharon',
+                           photo: "https://images.unsplash.com/photo-1667255949472-2f8bba82deb1?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=687&q=80\n  ",
+                           bio: 'oooooooo',
+                           postscounter: 0)
```
Comment (andersonlebon) [inline spec/intergration/post_index_spec.rb:9]: **In your post index page add**

- [ ] Tests that asserts you can see the user's profile picture.
- [ ] Tests that asserts you can see the first comments on a post.
- [ ] Tests that asserts you can see a section for pagination if there are more posts than fit on the vie

File: spec/intergration/user_show_spec.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe 'Users index', type: :system do
+  describe 'users#show page' do
+    before(:each) do
+      @user1 = User.create(name: 'Sharon',
```
Comment (andersonlebon) [inline spec/intergration/user_show_spec.rb:6]: **In your pshow page add**
- [ ] Tests that redirects to the post's show page when you click on a user's post.
- [ ] Tests that redirects to the user's post's index page when you click on a **see all posts**.

**Review by DeliceLydia — APPROVED** [review]
## Hi @businesskaniki, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 



### Cheers!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — COMMENTED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @businesskaniki, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**